core/conf/server_gpu_config.template

```diff
@@ -42,4 +42,5 @@ resource_config:
   search_resources:                 # define the devices used for search computation, must be in format: cpu or gpux
     - cpu
     - gpu0
-  index_build_device: gpu0          # CPU / GPU used for building index, must be in format: cpu or gpux
+  index_build_device:              # CPU / GPU used for building index, must be in format: cpu or gpux
```

Comment:
index_build_resources:              # CPU or GPUs used for building index, must be in format: cpu or gpux